allofplos/tests/test_unittests.py

```diff
@@ -39,11 +34,6 @@ def test_doi_conversions(self):
         self.assertEqual(example_file2, doi_to_path(example_doi2, ''), "{0} does not transform to {1}".format(example_doi2, example_file2))
```

Comment:
It would be better to make this a template string given how often you're using this pattern.